lib/helpers.js

```diff
@@ -17,17 +17,17 @@ export default function check(
       if (Date.now() - startTime > 10000) {
         return '[]';
       }
-      // Check for the common flow status messages and ignore them
-      if (errorM.indexOf('rechecking') !== -1 ||
-        errorM.indexOf('launching') !== -1 ||
-        errorM.indexOf('processing') !== -1 ||
-        errorM.indexOf('starting') !== -1 ||
-        errorM.indexOf('spawned') !== -1 ||
-        errorM.indexOf('logs') !== -1 ||
-        errorM.indexOf('initializing') !== -1
-      ) {
-        return check(pathToFlow, args, options);
-      }
+      // // Check for the common flow status messages and ignore them
```

Comment:
Just delete the code, if it's something that needs to come back the commit can be reverted.